feature-libs/customer-ticketing/components/customer-ticketing-reopen/customer-ticketing-reopen-dialog/customer-ticketing-reopen-dialog.component.ts

```diff
@@ -35,6 +36,23 @@ export class CustomerTicketingReopenDialogComponent implements OnInit {
       ?.allowedTypes;
   }
 
+  get getInputCharactersLimit(): number {
+    return (
+      this.customerTicketingConfig.customerTicketing?.inputCharactersLimit ||
+      2000
```

Comment:
How abut a constant to replace 2000 like MAX_INPUT_CHARACTERS

Reply:
yes, better.